tools/bin/ariaDownload.py

```diff
@@ -264,6 +261,40 @@ def make_nc_cookie(self):
        log.info('\n\nNew users: you must first log into Vertex and accept the EULA. In addition, your Study Area must be set at Earthdata https://urs.earthdata.nasa.gov')
        os.sys.exit(1)
 
+def prod_dl(nt):
+    """ Perform downloading using ASF bulk dl; parallel processing supported """
+    import multiprocessing
+    import ASFDataDload as AD
+    downloader  = AD.bulk_downloader()
+    max_threads = multiprocessing.cpu_count()
+
+    if nt == 'all':
+        nt = max_threads
+    else:
+        nt = int(nt)
+
+    nt = nt if nt < max_threads else max_threads
+    log.info('Using  %s threads for parallel downloads', nt)
+    chunks = np.array_split(downloader.files, nt)
+    chunks = [chunk for chunk in chunks if chunk.size > 0]
+    chunk_check = []
+    # sanity check
+    for chunk in chunks: chunk_check.extend(chunk)
+    assert chunk_check == downloader.files, 'Chunking failed'
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/gh/aria-tools/ARIA-tools/pullRequest?prid=6122609)